ipatests/test_integration/test_sysaccounts.py

```diff
@@ -427,3 +474,607 @@ def test_system_account_modify_password(self):
+        assert_func(expected, got)
+
+
+class TestSystemAccountsBackupRestore(IntegrationTest):
+    """Integration tests for system accounts with backuprestore"""
+
+    topology = "line"
+    num_clients = 0
+    num_replicas = 0
+    sys_acc1 = "account-1"
+    sys_acc2 = "account-2"
+    sys_acc3 = "account-3"  # Disabled account
+    desc1 = "System Account 1"
+    desc2 = "System Account 2"
+    desc3 = "System Account 3"
+    privilege_name1 = "test-privilege-password-privilege1"
+    privilege_name2 = "test-privilege-password-privilege2"
+    permission = "System: Change User password"
+    role_name1 = "test-privilege-role1"
+    role_name2 = "test-privilege-role2"
+    password = "Secret123"
+
+    @classmethod
+    def install(cls, mh):
+        """Install and set up all system accounts for backup/restore tests"""
+        super(TestSystemAccountsBackupRestore, cls).install(mh)
+        tasks.kinit_admin(cls.master)
+
+        # Setup privileged system account
+        setup_sysaccout(
+            cls.master, cls.sys_acc1, cls.privilege_name1,
+            cls.permission, cls.role_name1, privileged=True,
+            desc=cls.desc1, password=cls.password
+        )
+
+        # Setup unprivileged system account
+        setup_sysaccout(
+            cls.master, cls.sys_acc2, cls.privilege_name2,
+            cls.permission, cls.role_name2, privileged=False,
+            desc=cls.desc2, password=cls.password
+        )
+
+        # Setup disabled system account
+        setup_sysaccout(
+            cls.master, cls.sys_acc3, f"{cls.sys_acc3}-privilege",
+            cls.permission, f"{cls.sys_acc3}-role", privileged=False,
+            desc=cls.desc3, password=cls.password
+        )
+        cls.master.run_command(
+            ["ipa", "sysaccount-disable", cls.sys_acc3]
+        )
+
+    def test_system_account_data_backup_and_restore(self):
+        """Test data-only backup and restore with sysaccounts"""
+        sysaccount_name = "data-backup-test"
+        with restore_checker(self.master, self):
+            backup_path = tasks.get_backup_dir(self.master, data_only=True)
+
+            # Create sysaccount after backup
+            self.master.run_command(
+                [
+                    "ipa", "sysaccount-add", sysaccount_name,
+                    "--desc", "Created after backup",
+                    "--random"
+                ]
+            )
+            # Restore from backup
+            tasks.ipa_restore(self.master, backup_path)
+
+            # Verify sysaccount created after backup is removed by restore
+            tasks.wait_for_sssd_domain_status_online(self.master)
+            tasks.kinit_admin(self.master)
+            result = self.master.run_command(
+                ["ipa", "sysaccount-show", sysaccount_name],
+                raiseonerr=False
+            )
+            assert "not found" in result.stderr_text.lower()
+
+    def test_system_account_backup_restore_disabled(self):
```

Comment:
You can remove this test as there is already a check for the disabled sysaccount.